docgen/plugins/helpers.js

```diff
@@ -13,6 +13,12 @@ export default function helpers(filenames, metalsmith, cb) {
       const lang = opts && opts.lang;
       return highlight(src, lang);
     },
+    maybeActive(navPath, singlePathOrArrayOfPaths) {
+      const pathsToTest = [].concat(singlePathOrArrayOfPaths);
```

Comment:
I had to test this in console to actually enaure its doing what I want eheh but yes funn